src/mozanalysis/experiment.py

```diff
@@ -629,12 +657,18 @@ def _build_enrollments_query(
                 raise ValueError(
                     "App ID must be defined for building Cirrus enrollments query"
                 )
+            if not self.experimental_unit == ExperimentalUnit.CLIENT:
+                raise IncompatibleExperimentalUnit(
+                    "Cirrus enrollments currently only support client_id analysis units"
+                )
```

Comment:
So should we just ignore the `experimental_unit` here? Or is this intended to prevent an invalid configuration where a Cirrus experiment has an `experimental_unit` when it shouldn't?

Reply:
My thinking was to be explicit and fail if the experimental unit was used anywhere other than desktop experiments (normandy enrollments). Alternatively, we could ignore it (possibly logging a warning). My concern is that misconfiguration here could be subtle (there wouldn't be any way to directly tell from the results whether the configuration was applied), so if someone tries to do the wrong thing, they might think that it worked, which would be confusing.